enterprise/models.py

```diff
@@ -65,6 +65,7 @@ class Meta(object):
 
     uuid = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Enterprise Customer name."))
+    catalog = models.PositiveIntegerField(null=True, help_text=_("Course catalog for the Enterprise Customer."))
```

Comment:
Yeah, the course discovery API doesn't support querying catalogs by name, so we have to save the ID. I could see if I could find a way to save the name as well, but that seems a bit unnecessary given the admin UI will handle getting it for us.